packages/traits/reservoir_turbine_prototype_from.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
Sorry, needed: last print is of the form "print(a, b)".

Removing my erroneous comment.